src/stories/other.stories.tsx

```diff
@@ -21,6 +25,24 @@ stories.addDecorator(withKnobs)
         height: '100vh',
     }));
 
+stories.add('Ratings', withInfo(
+    'Ratings component'
+)(() =>
+    <div style={{ width: 200 }} >
+        <Ratings max={5} rating={4.3} Icon={HighlightIcon} />
+
+        <Ratings max={10} rating={4.3} Icon={HighlightIcon} />
+
+        <Ratings max={10} rating={0} Icon={HighlightIcon} />
+
+        <Ratings max={5} rating={3.5} Icon={ApprovedIcon} />
+
+        <Ratings max={5} rating={5 / 3} Icon={ApprovedIcon} />
+
+        <Ratings max={5} rating={null} Icon={ApprovedIcon} />
```

Comment:
Instead of having these multiple times, it would be  better to use a knob, so that the user can set the props themselves through storybook